### ISSUE TITLE
Update README and code documentation for JGI file staging

### ISSUE THREAD

**Issue by mbthornton-lbl** [microbiomedata/nmdc_automation#84]
Update the README with the details for:
- System setup
- Required data files
- Detailed steps

Improve documentation in the code

**Comment by ssarrafan** [comment]
@mbthornton-lbl I think you mean to add this to the new sprint so I'm moving it over.

**Comment by mflynn-lanl** [comment]
This is waiting on PR for #444 to be approved and merged into main

**Comment by ssarrafan** [comment]
@mflynn-lanl there are reviews waiting in #444 
I'll move this one more sprint since you're OOO today
FYI @aclum

**Comment by ssarrafan** [comment]
@mflynn-lanl what's the status of this?

**Comment by mflynn-lanl** [comment]
I will update the readme now that #444 has been merged

**Comment by ssarrafan** [comment]
Removing this from the sprint board since it hasn't been updated since August @mflynn-lanl @aclum

**Comment by jasmynp** [comment]
Hi @ssarrafan, no PR. Yes, next sprint sounds good.

**Comment by jasmynp** [comment]
Need to work through code before I can make useful updates here. Putting on the backlog for now.
